main/ertp/api/purse.md

```diff
@@ -53,7 +86,7 @@ const quatloosPurse.deposit(quatloos5);
 const currentBalance = quatloosPurse.getCurrentAmount();
 ```
 
-## purse.getCurrentAmountNotifier()
+## purse.getCurrentAmountNotifier() DEPRECATED AS OF 2021-1-1. See purse.getRecentAmountNotifier()
```

Comment:
Hmm, is there a reason to say 2021-1-1 rather than today's date?

Reply:
Didn't know when the code changes would go into effect, so figured a future date was better, and 1-1  seemed reasonable given the lack of stuff going on in general this week.